js/main.js

```diff
@@ -0,0 +1,59 @@
+const NAMES = ['Олег', 'Алина', 'Ваня', 'Юра', 'Никита', 'Алиса', 'Маша', 'Кирилл', 'Павел',
+  'Вадим', 'Соня', 'Лев', 'Игорь'];
+
+const MESSAGES = ['Всё отлично!', 'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'];
+
+const DESCRIPTIONS = ['Мое фото', 'Удачный кадр', 'мммммммм', '😜', '😈', '😎', '🤡', '🤢', '🤮', '🤬',
+  '☝🏿', 'записывайтесь на ноготочки 💅', '+ 10 社会学分', 'cringe', 'Biergetränk wärmt die Seele',
+  '✝', '🍆💦'];
+
+const NUMBER_OF_GENERATED_PHOTOS = 25;
+
+const getRandomNumber = (min, max) => {
+  const lower = Math.ceil(Math.min(Math.abs(min), Math.abs(max)));
+  const upper = Math.floor(Math.max(Math.abs(min), Math.abs(max)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+
+  return Math.floor(result);
+};
+
+const createRandomIdFromRange = (min, max) => {
```

Comment:
Круто, что умеешь работать с замыканиями!)

Reply:
Давай немного поменяем нейминг, потому что сейчас, например, в переменной `photoIdGenerator` лежит функция и мы не выполняем критерий по именованию перменных

```suggestion
const getRandomIdCreator = (min, max) => {
```